stories/buglink.stories.js

```diff
@@ -0,0 +1,19 @@
+import BugLink from "../src/components/BugLink.svelte";
+
+export default {
+  title: "Bug Links",
+};
+
+export const Bugzilla = () => ({
+  Component: BugLink,
+  props: {
+    ref: "https://bugzilla.mozilla.org/1234",
+  },
+});
+
+export const GitHub = () => ({
```

Comment:
I noticed that this component name was rendered as "Git Hub" instead of "Github" in Storybook. I think it's just a Storybook bug.

Reply:
Right, this is probably just how storybook interprets camel case. I tried to see if there was a way to override the title for a subcomponent, but didn't find one offhand. I think let's just leave this for now, since it doesn't impact the UX of the actual app. :)